integrations/slack/internal/vars/vars.go

```diff
@@ -15,6 +15,11 @@ type Vars struct {
 var (
 	AuthType = sdktypes.NewSymbol("authType")
 
+	// Custom OAuth
+	ClientID         = sdktypes.NewSymbol("clientID")
+	ClientSecret     = sdktypes.NewSymbol("clientSecret")
+	SigningSecret    = sdktypes.NewSymbol("signingSecret")
```

Comment:
I know the casing you're using here is consistent with the other vars, but I want to slowly migrate to `snake_case`, for all vars. Please use this in the new vars to avoid future migrations.

Reply:
Done